package-lock.json

```diff
@@ -165,6 +165,12 @@
                 "@types/node": "*"
```

Comment:
nit: can you prune this, it seems like it's not deleting vscode related things

Reply:
I don't understand, what should be removed?

Reply:
the vscode import was removed, but nothing was removed from package-lock which seems weird